app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Hmm - in my experience we usually have a single .gitignore file in the root folder

Reply:
Generated ones, like the other project but I agree with you.